sync/class.jetpack-sync-defaults.php

```diff
@@ -311,7 +311,7 @@ static function get_max_sync_execution_time() {
 	static $default_post_meta_whitelist = array();
 	static $default_comment_meta_whitelist = array();
 	static $default_disable = 0; // completely disable sending data to wpcom
-	static $default_sync_via_cron = 0; // use cron to sync
+	static $default_sync_via_cron = 1; // use cron to sync
```

Comment:
From our chats today, it seems best to move forward with sync via cron as the default. This will allow us to test the alternative on sites where we know cron is broken.